MonoGame.Framework/Input/GamePad.SDL.cs

```diff
@@ -122,39 +122,40 @@ private static GamePadCapabilities PlatformGetCapabilities(int index)
             if (!Gamepads.ContainsKey(index))
                 return new GamePadCapabilities();
 
-            if (Sdl.GameController.GetName(Gamepads[index].Device) == "Unknown Gamepad")
-                return new GamePadCapabilities
-                {
-                    IsConnected = true
-                };
+            var gamecontroller = Gamepads[index].Device;
+            var name = Sdl.GameController.GetName(gamecontroller);
+            var id = Sdl.Joystick.GetGUID(Sdl.GameController.GetJoystick(gamecontroller)).ToString();
+            var hasbutton = (name != "Unknown Gamepad");
 
             return new GamePadCapabilities
             {
                 IsConnected = true,
-                HasAButton = true,
-                HasBButton = true,
-                HasXButton = true,
-                HasYButton = true,
-                HasBackButton = true,
-                HasStartButton = true,
-                HasDPadDownButton = true,
-                HasDPadLeftButton = true,
-                HasDPadRightButton = true,
-                HasDPadUpButton = true,
-                HasLeftShoulderButton = true,
-                HasRightShoulderButton = true,
-                HasLeftStickButton = true,
-                HasRightStickButton = true,
-                HasLeftTrigger = true,
-                HasRightTrigger = true,
-                HasLeftXThumbStick = true,
-                HasLeftYThumbStick = true,
-                HasRightXThumbStick = true,
-                HasRightYThumbStick = true,
-                HasLeftVibrationMotor = true,
-                HasRightVibrationMotor = true,
-                HasVoiceSupport = true,
-                HasBigButton = true
+                DisplayName = name,
+                Identifier = id,
+                HasAButton = hasbutton,
+                HasBButton = hasbutton,
+                HasXButton = hasbutton,
+                HasYButton = hasbutton,
+                HasBackButton = hasbutton,
+                HasStartButton = hasbutton,
+                HasDPadDownButton = hasbutton,
+                HasDPadLeftButton = hasbutton,
+                HasDPadRightButton = hasbutton,
+                HasDPadUpButton = hasbutton,
+                HasLeftShoulderButton = hasbutton,
+                HasRightShoulderButton = hasbutton,
+                HasLeftStickButton = hasbutton,
+                HasRightStickButton = hasbutton,
+                HasLeftTrigger = hasbutton,
+                HasRightTrigger = hasbutton,
+                HasLeftXThumbStick = hasbutton,
+                HasLeftYThumbStick = hasbutton,
+                HasRightXThumbStick = hasbutton,
+                HasRightYThumbStick = hasbutton,
+                HasLeftVibrationMotor = hasbutton,
+                HasRightVibrationMotor = hasbutton,
+                HasVoiceSupport = hasbutton,
+                HasBigButton = hasbutton
```

Comment:
@cra0zy - Could we in a different PR make improvements here and actually query from SDL what buttons a gamepad has available?